### PR TITLE
chore: renamed 'tag schema' keyword to 'meshTag'

### PR DESCRIPTION
**Pull request by kshitijyelpale**
Concerns CU-bwzzg4

### CODE REVIEW

**Comment by meshcloud-gmbh** [comment]
Task linked: [CU-bwzzg4 Cleanup tag schema migration & YML configuration](https://app.clickup.com/t/bwzzg4)

File: docs/meshcloud.payment-methods.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ One way of creating payment methods is via the meshPanel. To do so, make sure th
 5. Choose a type for the payment method. Read more [here](#types-of-payment-methods) on what type to choose and what the individual differences are.
 6. (This is optional) Set a maximum amount of EUR on the payment method to indicate the remaining budget of this payment method.
 7. (This is optional) Set an expiration date for the payment method. This is especially useful when a budget expires, e.g. at the end of the accounting year.
-8. Additionally, you can enter tags for the payment method, which are custom per meshImplementation (also see [Tag Schema](meshstack.metadata-tags.md)). This is useful when you want to enhance the payment method with organizational details like the cost center number or the business unit.
+8. Additionally, you can enter tags for the payment method, which are custom per meshImplementation (also see [meshTag](meshstack.metadata-tags.md)). This is useful when you want to enhance the payment method with organizational details like the cost center number or the business unit.
```
Comment (Jelledb) [inline docs/meshcloud.payment-methods.md:35]:
```suggestion
8. Additionally, you can enter tags for the payment method, which are custom for your meshStack (also see [meshTags](meshstack.metadata-tags.md)). This is useful when you want to enhance the payment method with organizational details like the cost center number or the business unit.
```

File: docs/meshstack.billing-configuration.md [hunk 2]
```diff
@@ -348,7 +348,7 @@ The following metadata keys are derived from meshStack metadata and therefore av
 
 ### Available tag keys
 
-Custom tags can be referenced via their property name in the according [tag schema](meshstack.metadata-tags.md#tag-schemas).
+Custom tags can be referenced via their property name in the according [meshTag](meshstack.metadata-tags.md).
```
Comment (Jelledb) [inline docs/meshstack.billing-configuration.md:351]: Please remove the marked section and replace it with something like

```
It is also possible to derive [meshTags](meshstack.metadata-tags.md) as metadata keys by using its tag name.
```

**Review by Jelledb — APPROVED** [review]
👌

File: docs/meshstack.billing-configuration.md (hunk 2 above)
Comment (kshitijyelpale) [reply docs/meshstack.billing-configuration.md]: done.

**Pull request merged**